docker-compose.test-parallel-local.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.7'
```

Comment:
Maybe there is a way to inherit from a common docker compose here no? To factorize with the CI one

Reply:
Done, docker-compose.test-parallel-base.yml is now the common compose which is inherited for ci and local use